app/Frame.js

```diff
@@ -3,6 +3,110 @@ import { AppContext } from "./context/AppContext";
 import { BrowserRouter, Route, Switch } from "react-router-dom";
 import { useMediaPredicate } from "react-media-hook";
 
+/* CLDR Data */
```

Comment:
Add to barrel file so that there is only one import here

Reply:
Done.